webpack.config.js

```diff
@@ -37,35 +37,35 @@ let outputPath;
 let devtool;
 
 var getConfig = function () {
-	  var config;
-
-	  try {
-	    // look for config file
-	    config = require('./config.json');
-	  } catch (err) {
-	    // create file with defaults if not found
-	    config = {
-	      'LOCAL_OWA_FOLDER': '/Users/your-user/referenceapplication-standalone-2.8.0/appdata\\owa/',
-	      'APP_ENTRY_POINT': 'http://localhost:8081/openmrs/owa/openmrs-owa-labworkflow/index.html'
-	    };
-
-	    fs.writeFile('config.json', JSON.stringify(config));
-
-	  } finally {
-	    return config;
-	  };
-	}
+	var config;
```

Comment:
We seem to use 2 space indentation across the codebase, not sure why this file indentation was changed to 4.

Reply:
@jomadoye good catch! my editor has a mind of it's own and sometimes I find myself changing it to 2 but I don't notice when it changes back to 4, do you think we'll need to set an editor config since we both use vscode ?